src/popup/router/pages/Names/Details.vue

```diff
@@ -73,11 +74,13 @@ export default {
       return chekAensName(this.pointer) || checkAddress(this.pointer);
     },
     hasPointer() {
-      return getAddressByNameEntry(this.nameEntry);
+      return this.nameEntry.pointers.accountPubkey;
     },
   },
-  created() {
+  async created() {
+    await this.$watchUntilTruly(() => this.sdk);
     this.$store.dispatch('names/fetchOwned');
+    this.nameHash = (await this.sdk.api.getNameEntryByName(this.name)).id;
```

Comment:
Would be more accurate to refetch the hash each time `this.name` changes, proposing to define this in the `watch` section.